.circleci/config.yml

```diff
@@ -425,21 +425,15 @@ jobs:
     executor: ubuntu-machine-executor
     steps:
       - checkout
-      - when:
-          condition: &deploy-condition
-            or:
-              - equal: [main, << pipeline.git.branch >>]
-              - << pipeline.git.tag >>
```

Comment:
Are we sure we do not want to do a deploy when a new tag is introduced? I guess the question is, do we even tag things?

Reply:
I'll defer to @relud on this but I think the idea was tagged releases for this repo at some point

Reply:
yeah, this was built with tagging in mind, but we don't use them